.env.ci

```diff
@@ -1,3 +1,5 @@
 # Docker MongoDB URI
 MONGODB_URI=mongodb://root:root@auth_mongo:27017/auth?authSource=admin&ssl=false
 MONGODB_URI_TEST_OVERRIDE=mongodb://root:root@auth_mongo:27017/auth_test?authSource=admin&ssl=false
+AIRTABLE_API_KEY=xxxxxxxxxxxxxxxxx
```

Comment:
You don't need to replicate vars from `.env.example` in `.env.ci`. The `.env.ci` file is loaded in addition to the `.env.example` file during CI runs.